EasyVCR/InternalUtilities/JSON/Serialization.cs

```diff
@@ -110,14 +111,22 @@ internal static string ConvertObjectToJson(object obj, IOrderOption orderOption,
         ///     Normalize a JSON string to remove CRLF and other whitespace.
         /// </summary>
         /// <param name="json">JSON string to normalize.</param>
+        /// <param name="removeElements">List of elements to remove from the JSON string.</param>
         /// <returns>Normalized JSON string.</returns>
-        internal static string? NormalizeJson(string json)
+        internal static string? NormalizeJson(string json, List<CensorElement>? removeElements = null)
         {
             if (string.IsNullOrWhiteSpace(json))
             {
                 return null;
             }
 
+            // need to use censors to remove elements from the JSON string
+            if (removeElements != null)
+            {
+                return Censors.CensorJsonData(json, "FILTERED", removeElements);
```

Comment:
Prefer `REDACTED`

Reply:
This is just used ephemerally to normalize elements during matching. It's value doesn't actually matter.